pkg/sentry/socket/netstack/provider.go

```diff
@@ -126,6 +126,12 @@ func (p *provider) Socket(t *kernel.Task, stype linux.SockType, protocol int) (*
 		ep, e = eps.Stack.NewRawEndpoint(transProto, p.netProto, wq, associated)
 	} else {
 		ep, e = eps.Stack.NewEndpoint(transProto, p.netProto, wq)
+
+		// Assign task to PacketOwner interface to get the UID and GID for
+		// iptables owner matching.
+		var owner tcpip.PacketOwner
+		owner = t
+		ep.SetOwner(owner)
```

Comment:
This can all be collapsed into a single line, Go will figure out that t implements PacketOwner:
```go
ep.SetOwner(t)
```